pkg/asset/manifests/powervs/cluster.go

```diff
@@ -175,7 +175,12 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 			return nil, fmt.Errorf("unable to find a DNS server for specified VPC: %s %w", installConfig.Config.PowerVS.VPCName, err)
 		}
 
-		powerVSCluster.Spec.DHCPServer.DNSServer = &dnsServerIP
+		powerVSCluster.Spec.DHCPServer = &capibm.DHCPServer{
+			Cidr: &dhcpSubnet,
```

Comment:
Nit: Can we add a comment about adding the subnet information here too for commit purposes.

Reply:
@barbacbd this is actually so I use the same subnet specified above but this makes me realize that I went about this wrong. Reinstantiating the struct is not the right move. I should update it if possible.

Will recommit in a moment.